rhombus/rhombus/tests/array.rhm

```diff
@@ -183,6 +187,15 @@ check:
   def Array(0, _, ...) = Array(0, 1, 2)
   ~completes
 
+check:
+  def Array(0, _, ...) = Array(0)
+  ~completes
+
+check:
+  def Array(0, _, ... ~nonempty) = Array(0)
+  "ok"
+  ~raises error.annot_msg()
```

Comment:
`~raises` → `~throws`, here and elsewhere

Reply:
Ah — missed this before I merged. I'll push a repair.